portal-ui/src/screens/Console/License/License.tsx

```diff
@@ -81,26 +80,29 @@ const styles = (theme: Theme) =>
     detailsContainerBorder: {
       border: "1px solid #e2e2e2",
       borderBottom: 0,
+      borderRadius: "4px 4px 0px 0px",
     },
     detailsContainerBorderHighlighted: {
-      border: "1px solid #9a93ad",
+      border: "1px solid #B5B5B5",
       borderBottom: 0,
     },
     detailsTitle: {
-      fontSize: 17,
+      fontSize: 19,
       fontWeight: 700,
       marginBottom: 26,
       paddingTop: 18,
     },
     currentPlan: {
       fontWeight: 700,
-      background:
-        "transparent linear-gradient(90deg, #073052 0%, #081C42 100%) 0% 0% no-repeat padding-box",
-      boxShadow: "0px 3px 7px #00000014",
-      color: "#fff",
+      background: "#D5DDE5",
+      borderRadius: "3px 3px 0px 0px",
+      color: "#121212",
+      padding: "8px",
```

Comment:
Please leave it as int
```suggestion
      padding: 8,
```